aspnet/2-structured-data/Models/DatastoreBookStore.cs

```diff
@@ -29,6 +35,8 @@ public static class DatastoreBookStoreExtensionMethods
         /// <returns>A datastore key.</returns>
         public static Key ToKey(this long id)
         {
+            // P0 This function is especially useful, but I see no way to implement it

```

Comment:
That's a good question - *maybe* it is. I'll have to experiment. When I'm back from vacation, I'll talk to the Datastore team about it, so that the docs can be clear one way or the other.